mitodl/hugo-course-publisher
# update ocw-to-hugo commands & lock version

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/hugo-course-publisher/issues/200

#### What's this PR do?
updates the `import:*` commands to support the changes made to `ocw-to-hugo` that make its output mirror the `site/content` directory.

#### How should this be manually tested?
Run `yarn install` and verify that you have `ocw-to-hugo@1.0.3`.  Then, run `npm run import:ocw:example_courses` and assure that the content structure is correct and that the site renders correctly with `npm start`.
